demo/actions.py

```diff
@@ -662,14 +670,29 @@ def name(self) -> Text:
         return "action_next_step"
 
     def run(self, dispatcher, tracker, domain) -> List[EventType]:
-        step = int(tracker.get_slot("step")) + 1
+        if tracker.get_slot("step"):
+            step = int(tracker.get_slot("step")) + 1
 
-        if step in [2, 3, 4]:
-            dispatcher.utter_message(template=f"utter_continue_step{step}")
-        else:
-            dispatcher.utter_message(template="utter_no_more_steps")
+            if step in [2, 3, 4]:
+                dispatcher.utter_message(template=f"utter_continue_step{step}")
+            else:
+                dispatcher.utter_message(template="utter_no_more_steps")
 
-        return []
+            return []
+
+        else:
+            # trigger get_started_step1 intent
+            return [
+                ActionExecuted("action_listen"),
+                SlotSet("step", "1"),
+                UserUttered(
+                    "/get_started_step1",
+                    {
+                        "intent": {"name": "get_started_step1", "confidence": 1.0},
+                        "entities": {},
```

Comment:
How does the step slot get set usually? Would it normally happen after the /get_started_step1 intent?

Reply:
Normally it happens like this:

```
* get_started_step1
    - action_greet_user
```

The custom action `action_greet_user` then sets the step:

```
    elif intent[:-1] == "get_started_step" and shown_privacy:
            dispatcher.utter_message(template=f"utter_{intent}")
            return [SlotSet("step", intent[-1])]
```

sooooo, I think I do not need to set the step myself. It should happen automatic. Let me test this out.

Reply:
Hm maybe the action sets it, it just isn't featurized and therefore isn't in the stories? haven't looked too deeply into the stories/actions.

Reply:
@erohmensing ,
I can indeed remove it, and then the slot `step` will be set as usual by the `action_greet_user`.

Good catch!